alembic/script/revision.py

```diff
@@ -964,6 +1034,11 @@ def verify_rev_id(cls, revision):
     def __init__(
         self, revision, down_revision, dependencies=None, branch_labels=None
     ):
+        if down_revision and revision in down_revision:
```

Comment:
I've hit an error that I think is related to this change (#784). I think this will incorrectly flag a self-loop if `revision` is a substring of `down_revision`.

Should this instead be:
```python
if down_revision and revision == down_revision:
```

Reply:
yes.  it might be a tuple also.  if you'd like to review the fix see https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2492